class.jetpack-options.php

```diff
@@ -404,4 +414,27 @@ static function get_raw_option( $name, $default = null ) {
 		return $value;
 	}
 
+	/**
+	 * This function lets us by pass certain options to be gotten via the raw sql calls.
```

Comment:
I think this needs better English, cc @roccotripaldi could you help us here?

Reply:
"This function checks for a constant that, if present, will disable direct DB queries Jetpack uses to manage certain options and force Jetpack to always use Options API instead.

Options can be selectively managed via a blacklist by filtering option names via the `jetpack_disabled_raw_option` filter."